### DISCUSSION TITLE
first item of grid or flex is bigger than other ?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#4523] [Help]
Hi let's take a look at images. I can't understand why the first item not equal other @@
I tried both grid and flex. 
https://i.imgur.com/CgG7LVx.gif

![image](https://user-images.githubusercontent.com/15695252/120248530-c8d72c00-c2a1-11eb-8c41-913b2a71e614.png)

**Comment by simonswiss** [comment]
Hey!

Are you able to reproduce this in a Tailwind Play example?

https://play.tailwindcss.com

If not, perhaps a public GitHub repo? It's hard to troubleshoot from a screen recording.

**Comment by janldeboer** [comment]
[Here is an example](https://play.tailwindcss.com/tGIoDtjXtB)
The ```space-x-``` causes the trouble
I currently face the same issue. I guess padding/margin can be used instead, but are only a workaround in my opinion.

**Reply by onmax** [reply]
Hey, you can use `gap-x-` class instead. See https://play.tailwindcss.com/r2dwE2RK2j

You can learn more in the official doc 😃: https://tailwindcss.com/docs/space#limitations

**Reply by janldeboer** [reply]
Perfect, thank you!
